README.md

```diff
@@ -369,6 +388,30 @@ The `iotedgedev` module has the following commands:
 `iotedgedev project --help`
 - `--create TEXT`  Creates a new Azure IoT Edge project. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.
 
+
+**azure**
+
+`iotedgedev azure --help`
+
+- `--setup`                    Reads the required Azure components
+                               configuration from your subscription. Creates
+                               new Azure resources or uses existing ones.
+                               [required]
+- `--azure-credentials TEXT TEXT`  The credentials (username password) to use to
+                               login to Azure. Leave empty to login in
+                               interactive mode.
+- `--subscription TEXT`        The Azure subscription name or id to use.
+                               [required]
+- `--resource-group-name TEXT` The name of the new Resource Group to use.
+                               [required]
+- `--iothub-name TEXT`         The IoT Hub name to be used. Creates a new IoT
+                               Hub if not found.  
+                               [required]
+- `--edge-device-id TEXT`      The IoT Edge Device Id to use or create.
+                               [required]
+- `--update-dotenv`            If set, the current .env will be updated with
```

Comment:
Just confirming.  This honors the DOTENV_FILE setting right?  It should.  So that it updates that file instead of hardcoded to .env.

Reply:
Correct, but has a side effect, it removes all comments from the file. This is why it's off by default.

Reply:
I'm okay with off by default, just want to honor the DOTENV_FILE